src/main/java/org/ibp/api/ibpworkbench/rest/BreedingView.java

```diff
@@ -102,16 +112,27 @@ public DataResponse saveSsaResultSummary(
 			} else {
 				response = new DataResponse(false, "Errors invoking web service: " + errors);
 			}
-		} catch (Exception e) {
+		} catch (final Exception e) {
 			BreedingView.LOG.debug(e.getMessage(), e);
 			response = new DataResponse(false, "Failed to invoke service: " + e.toString());
 		}
 
 		return response;
 	}
 
-	public void setBreedingViewService(BreedingViewService breedingViewService) {
+	@ApiOperation(value = "Count number of days before Breeding View license expires")
```

Comment:
Please move this into a new BVLicenseResource, this class does not make much sense, and will have to be reviewed later